app/core/less/nav.less

```diff
@@ -49,6 +49,4 @@
 
 .nav-right {
   float: right;
-
-  margin-right: -@nav-item-horizontal-padding;
-}
\ No newline at end of file
+}
```

Comment:
this change is moving the signup link to the top right corner. 
Is it expected to be there ?

Reply:
Not from this change, acceptable for now.
We will see later if we want another layout.